packages/node/src/node/ServerNode.ts

```diff
@@ -115,7 +115,7 @@ export class ServerNode<T extends ServerNode.RootEndpoint = ServerNode.RootEndpo
     /**
      * Perform a factory reset of the node.
      */
-    async factoryReset() {
+    override async erase() {
```

Comment:
Just to be sure ... no need to call super method? there is logic in which is not reflected here

Reply:
Right, the logic here encompasses what base class does